### PR TITLE
Feature | Inflation Rewards

### PR DESCRIPTION
**Pull request by iverc**

### CODE REVIEW

File: version [hunk 1]
```diff
@@ -1 +1 @@
-0.9.12
+0.9.14
```
Comment (banshee) [inline version:1]: The previous version was 0.9.12, should this be 13?

File: app/setup_handlers.go [hunk 2]
```diff
@@ -4,29 +4,51 @@ import (
...
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/upgrade/types"
+
+	dispensationtypes "github.com/Sifchain/sifnode/x/dispensation/types"
 )
 
-const upgradeName = "0.9.12"
+const upgradeName = "0.9.14"
+
+const distributionVersion = "0.9.14"
```
Comment (Jedi2002) [inline app/setup_handlers.go:13]: Not a problem , but any reason why we have two constants with the same value ?

**Review by Jedi2002 — APPROVED** [review]

File: app/setup_handlers.go (hunk 2 above)
Comment (iverc) [reply app/setup_handlers.go:13]: @Jedi2002 cause the `upgradeName` constant will be changing for next versions, while the dispensation is a one time event

File: version (hunk 1 above)
Comment (iverc) [reply version:1]: @banshee I think we have not made the previous deployment with fix update (was deployed to testnet, but not mainnet) and in that fix the version was changed from 0.9.12 to 0.9.13
https://github.com/Sifchain/sifnode/pull/2120
I think we should merge that PR first (also fixes integration tests), and then do this one. Does it sound good?
cc @intl-man @AustinoBombino @Jedi2002

Comment (intl-man) [reply version:1]: @iverc Agreed. #2120 has been merged.

**Review by intl-man — APPROVED** [review]

**Pull request merged**